automated install/basic-install.sh

```diff
@@ -669,7 +669,7 @@ installScripts() {
 }
 
 installConfigs() {
-  # Install the configs from /etc/.pihole to their various locations
+  # Install the configs from ${PI_HOLE_LOCAL_REPO} to their various locations
```

Comment:
Ya, I meant to say that `PI_HOLE_LOCAL_REPO` would be better in a comment than `${PI_HOLE_LOCAL_REPO}`.